### DISCUSSION TITLE
How can I remove left padding from nvim-dap-ui & neotest?

### DISCUSSION THREAD

**Discussion by geardy47** [LazyVim/LazyVim#1764] [Q&A] [answered]
I want to remove left padding from nvim-dap-ui & neotest. It appeared on its own after updating to latest lazyvim

`Neotest`
<img width="1768" alt="Screenshot 2023-10-17 at 10 44 18 AM" src="https://github.com/LazyVim/LazyVim/assets/23553059/3df8727d-2501-444b-9ae6-8f5f800a909b">

`DAP UI`
<img width="1768" alt="Screenshot 2023-10-17 at 10 44 35 AM" src="https://github.com/LazyVim/LazyVim/assets/23553059/f67c1bd9-287f-408a-abf3-e1a6f540c551">

**Comment by folke** [comment, marked as answer]
That's a bug. I'm almost done fixing it

**Reply by geardy47** [reply]
I've updated it and it's fixed, thanks
